lib/wallet/src/Cardano/Wallet/DB/Layer.hs

```diff
@@ -570,12 +562,15 @@ newDBFreshFromDBOpen wF ti wid_ DBOpen{atomically=atomically_} =
                 case findNearestPoint wal requestedPoint of
                     Nothing -> throw $ ErrNoOlderCheckpoint wid_ requestedPoint
                     Just nearestPoint ->
+                        let nearestSlot = case nearestPoint of
```

Comment:
```suggestion
                        let nearestSlotNo = case nearestPoint of
```

Could you use this name instead? This is more correct, as `type Slot = WithOrigin SlotNo`.